<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextWindow</class>
 <widget class="QDialog" name="TextWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soko4: Text window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_get">
       <property name="toolTip">
        <string>Get the current board and translate it into .soko format</string>
       </property>
       <property name="text">
        <string>Get level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_play">
       <property name="toolTip">
        <string>Copy the content into play area</string>
       </property>
       <property name="text">
        <string>Copy to playboard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_get_sol">
       <property name="text">
        <string>Get the solution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_apply_sol">
       <property name="text">
        <string>Apply the solution</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
